### ISSUE TITLE
don't expose UUID

### ISSUE THREAD

**Issue by N4melessGitter** [mozilla/testpilot-containers#812]
https://github.com/mozilla/testpilot-containers/blob/master/webextension/js/content-script.js#L27

```js
imageElement.src = browser.extension.getURL("/img/container-site-d-24.png");
```

problem: https://bugzilla.mozilla.org/show_bug.cgi?id=1372288

solution: `imageElement.src = 'data:image/png;base64,...`

**Comment by jonathanKingston** [comment]
Thanks for the reminder to fix this.

**Comment by N4melessGitter** [comment]
> Thanks for the reminder to fix this.

np, thanks for the quick fix.

My JS skills are not great so I'm sorry if this is a stupid question but I recently learned that each call to createObjectURL() should be later followed by a revokeObjectURL(). Is that not necessary or possible here?
https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL#Notes

I would like to test this extension but I'm on ESR and I've noticed that the Requirements information is inconsistent: the README.html says Firefox 51+, the manifest.json has `"strict_min_version": "51.0"` but the README.md says Firefox 53+. Which one is correct?
